src/generic/Matrix.jl

```diff
@@ -249,7 +254,7 @@ and zeroes elsewhere. The matrix space must contain square matrices.
 one(a::AbstractAlgebra.MatSpace) = check_square(a)(1)
 
 @doc Markdown.doc"""
-    one(a::AbstractAlgebra.MatSpace)
+    one(a::MatElem)
```

Comment:
This would change the documentation to be different to the actual function. That seems invalid.

Reply:
Hmm, we have one method `one(a::AbstractAlgebra.MatSpace)` as well as the method `one(a::MatElem)`. Then both documentations shouldn't point to `one(a::AbstractAlgebra.MatSpace)`? I'm sorry if there's something I am not understanding.

Reply:
I am not sure what happened here. When I checked earlier, it seemed that the proposed change to the docstring did not match the function it was documenting. Now it seems to match. I'm a bit confused by that. Obviously your proposed change is correct.